src/data/markdownpages/secondPage.mdx

```diff
@@ -0,0 +1,67 @@
+## Copyblog/yet-another-post.mdx: copy code to clipboard
```

Comment:
i'll eventually close out this pr but just commiting this so I can come back to it if we switch to mdx at some point